common/test/acceptance/tests/studio/test_studio_container.py

```diff
@@ -1136,3 +1137,136 @@ def test_common_problem_types_tab(self):
+                    self.unit_page1,
+                    self.unit_page2
+                )
+            )
+        )
+
+    def verify_move_opertions(self, operation, component_display_names_after_operation):
+        """
+        Verify move operations.
+
+        Arguments:
+            operation (str),                                  `move` or `undo move` operation
+            component_display_names_after_operation (dict)    display names of components after operation in source/dest
+        """
+        unit_page = self.go_to_unit_page(unit_name='Test Unit 1')
+        components = unit_page.displayed_children
+        self.assertEqual(len(components), 2)
+
+        components[0].open_move_modal()
+        self.move_modal_view.navigate_to_category(self.source_xblock_category, self.navigation_options)
+        self.assertEqual(self.move_modal_view.is_move_button_enabled, True)
+
+        self.move_modal_view.click_move_button()
+        self.container.verify_confirmation_message(self.message_move)
+        self.assertEqual(len(unit_page.displayed_children), 1)
+
+        if operation == 'move':
+            self.container.click_take_me_link()
+        elif operation == 'undo_move':
+            self.container.click_undo_move_link()
+            self.container.verify_confirmation_message(self.message_undo)
+
+        unit_page = ContainerPage(self.browser, self.unit_page2.locator)
+        components = unit_page.displayed_children
+        self.assertEqual(
+            [component.name for component in components],
+            component_display_names_after_operation
+        )
+
+    def test_move(self):
+        """
+        Test if we can move a component successfully.
+
+        Given I am a staff user
+        When I go to unit page in first section
+        Then I open the move modal
+        Then I navigate to unit in second section from within move modal
+        Then I see move button is enabled
+        Then I click on the move button
+        Then I see move operation successfull message
+        When I go to unit page in second section
+        Then I see move compoenent there
+        """
+        self.verify_move_opertions(
+            operation='move',
+            component_display_names_after_operation=['HTML 21', 'HTML 22', 'HTML 11']
+        )
+
+    def test_undo_move(self):
+        """
+        Test if we can undo move a component successfully.
+
+        Given I am a staff user
+        When I go to unit page in first section
+        Then I open the move modal
+        Then I click on the move button
+        Then I see move operation successfull message
+        When I clicked on undo move link
+        Then I verified that undo move operation is successfull
+        """
+        self.verify_move_opertions(
+            operation='undo_move',
+            component_display_names_after_operation=['HTML 11', 'HTML 12']
+        )
+
+    def test_a11y(self):
```

Comment:
@jzoldak @benpatterson If i decorate this test with `attr('a11y')` then this test fails always due to same [failure reason](https://github.com/edx/bok-choy/blob/master/bok_choy/page_object.py#L299). From the screenshot it looks like that outline page never finishes loading but i am unable to understand the reason of failure. Tests passes Without `attr('a11y')`. 

Any idea why the tests fails with `attr('a11y')`?